x-pack/plugins/actions/server/usage/task.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Logger, ISavedObjectsRepository } from 'kibana/server';
+import moment from 'moment';
+import {
+  RunContext,
+  TaskManagerSetupContract,
+  TaskManagerStartContract,
+} from '../../../task_manager/server';
+import { ActionTypeRegistry } from '../action_type_registry';
+import {
+  getTotalCount,
+  getInUseTotalCount,
+  getExecutionsCount,
+  getTotalInUseCountByActionTypes,
+  getTotalCountByActionTypes,
+  getExecutionsCountByActionTypes,
+} from './actions_telemetry';
+
+export const TELEMETRY_TASK_TYPE = 'actions_telemetry';
+
+export const TASK_ID = `Actions-${TELEMETRY_TASK_TYPE}`;
+
+export function initializeActionsTelemetry(
+  logger: Logger,
+  savedObjectsRepository: ISavedObjectsRepository,
+  actionTypeRegistry: ActionTypeRegistry,
+  taskManager: TaskManagerSetupContract,
+  taskManagerStart: TaskManagerStartContract
+) {
+  registerActionsTelemetryTask(
+    logger,
+    savedObjectsRepository,
+    actionTypeRegistry,
+    taskManager,
+    taskManagerStart
+  );
+}
+
+export function scheduleActionsTelemetry(logger: Logger, taskManager?: TaskManagerStartContract) {
+  if (taskManager) {
+    scheduleTasks(logger, taskManager);
+  }
+}
+
+function registerActionsTelemetryTask(
+  logger: Logger,
+  savedObjectsRepository: ISavedObjectsRepository,
+  actionTypeRegistry: ActionTypeRegistry,
+  taskManager: TaskManagerSetupContract,
+  taskManagerStart: TaskManagerStartContract
+) {
+  taskManager.registerTaskDefinitions({
+    [TELEMETRY_TASK_TYPE]: {
+      title: 'Actions telemetry fetch task',
+      type: TELEMETRY_TASK_TYPE,
+      timeout: '1m',
```

Comment:
I wonder if that timeout is long enough.  It should run pretty fast, I'd think, under "normal" amounts of stress.  Would it be dangerous to set it to something like `5m` _just in case_???